lale/lib/sklearn/simple_imputer.py

```diff
@@ -95,7 +95,7 @@ def transform_schema(self, s_X):
                     "description": 'When strategy == "constant", fill_value is used to replace all occurrences of missing_values',
                 },
                 "verbose": {
-                    "type": "integer",
+                    "anyOf": [{"type": "integer"}, {"type": "string"}],
```

Comment:
Can you please guard this with a version constraint, since it is only deprecated starting in sklearn 1.1?
```python
if sklearn.__version__ >= "1.1":
    ... customize_schema ...
```

Furthermore, instead of 
```python
"anyOf": [{"type": "integer"}, {"type": "string"}],
```
this should probably be
```
"anyOf": [{"type": "integer"}, {"enum": ["deprecated"]}],
```
because not all strings are allowed, only the string `"deprecated"`.

Reply:
Thank you for the comments, I have updated the code, please review it again.

Reply:
Thanks! The change looks good to me. I think you need an `import sklearn` at the top of simple_imputer.py to resolve the error message from static checks.

Reply:
Sure